src/app/modules/gb-data-selection-module/accordion-components/gb-export/gb-export.component.html

```diff
@@ -74,20 +74,35 @@ <h5>Previous Exports</h5>
     </p-column>
     <p-column>
       <ng-template pTemplate="header">
-        <span>Download</span>
       </ng-template>
       <ng-template let-job="rowData" pTemplate="body">
         <button type="button"
                 *ngIf="job.jobStatus === 'Completed'"
                 class="btn btn-outline-primary btn-sm"
+                [disabled] = "job.isInDisabledState"
                 (click)="downloadExportJob(job)">
           <span class="fa fa-download"></span>
           Download
         </button>
-
+        <button type="button"
+                *ngIf="['Completed','Error','Cancelled'].indexOf(job.jobStatus) < 0;else archive"
```

Comment:
You can use `includes` instead of `indexOf`. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes